kernel/names.mli

```diff
@@ -746,6 +731,29 @@ end
 
 type projection = Projection.t
 
+(** {6 Global reference is a kernel side type for all references together } *)
+
+(* XXX: Should we define GlobRefCan GlobRefUser? *)
```

Comment:
I see that  `eq_gr` uses canonical names. I find that a bit surprising. We should probably review the uses to know if equality modulo user names makes sense.

Reply:
Yup.